app/javascript/packs/application.js

```diff
@@ -50,6 +51,11 @@ document.addEventListener('DOMContentLoaded', () => {
       logClick(event)
     })
   }
+  if (document.getElementById('covid19-page-content')) {
```

Comment:
Would it help to track user flows through the COVID-19 advanced search UI?

Applying `logPageView` and `logClick` here would enable that.  Our Google Analytics fallback can provide basic event flows right now, while Mixpanel progresses upstream.